src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/ErrorReport.scala

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.dsde.workbench.leonardo.model
+
+import akka.http.scaladsl.model.StatusCode
+import spray.json._
+
+import scala.language.postfixOps
+
+/*
+ * THIS CLASS NEEDS TO BE REMOVED ASAP
```

Comment:
No, because:
1. rawlsModel isn't built for 2.12 / akka-http
2. rawlsModel has a bunch of rawls stuff in it, which is huge dependency overkill

I have a back-burner task to yank that stuff out into a new repo but it'll probably be a little while before that happens. (Unless sam does it first!)